### DISCUSSION TITLE
How do I install XFCE desktop?

### DISCUSSION THREAD

**Discussion by kiena0573** [ophub/amlogic-s9xxx-armbian#1610] [Amlogic]
i'm using Armbian_23.08.0_amlogic_s905x-t95_jammy_5.10.189_server_2023.08.09.img.gz. after flashing i dont find any DE nor DM. how do i properly install xfce? it's been frustrating trying and trying...

**Comment by ophub** [comment]
armbian-software
201

**Reply by a2sy** [reply]
When I tried it, it installed gnome instead of xfce.

**Reply by PJ-568** [reply]
There is a selection in `armbian-config` called `desktop setting`, change it to xfce before `armbian-software` `201`.

**Reply by jahidulislamjahid** [reply]
> There is a selection in `armbian-config` called `desktop setting`, change it to xfce before `armbian-software` `201`.

Hi bro, I have tried as you said but couldn't find the desktop setting in armbian config. is there any other way to install xfce?
I have tried to install xfce using sudo apt install xfce xfce-goodies also tried thrugh tasksel. it installs fine but fails to start lightdm. what should I do?

**Reply by jahidulislamjahid** [reply]
> > There is a selection in `armbian-config` called `desktop setting`, change it to xfce before `armbian-software` `201`.
> 
> Hi bro, I have tried as you said but couldn't find the desktop setting in armbian config. is there any other way to install xfce?
> I have tried to install xfce using sudo apt install xfce xfce-goodies also tried thrugh tasksel. it installs fine but fails to start lightdm. what should I do?

@ophub any solution to this bro? or I'm just stuck with Gnome?
